js/contextMenus.js

```diff
@@ -472,7 +472,7 @@ function tabTemplateInit (frameProps) {
         label: locale.translation('reloadTab'),
         click: (item, focusedWindow) => {
           if (focusedWindow) {
-            focusedWindow.webContents.send(messages.SHORTCUT_FRAME_RELOAD, frameKey)
+            CommonMenu.sendToFocusedWindow(focusedWindow, messages.SHORTCUT_FRAME_RELOAD, frameKey)
```

Comment:
I also see that you are passing `frameKey` and `tabId` into `CommonMenu.sendToFocusedWindow`. which only accepts `focusedWindow` and `message`

Reply:
Indeed. I was counting on ci to find any problems and it looks like this needs some rethinking

Reply:
I'll just remove the calls that pass extra params for now